spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender.py

```diff
@@ -347,6 +358,14 @@ def _transform(self, df: DataFrame) -> DataFrame:
                     json_df = df.sql_ctx.read.text(
                         path_to_files, pathGlobFilter="*.json", recursiveFileLookup=True
                     )
+                    if sort and sort.get("column_for_sorting"):
+                        json_df = json_df.withColumn(
+                            sort["column_for_sorting"],
+                            get_json_object(
+                                col("value"), f"$.{sort['column_for_sorting']}"
+                            ),
+                        ).orderBy(sort["column_for_sorting"])
+
```

Comment:
why do we need to orderBy here when we're sorting within partitions later?

Reply:
Oh, I'd add this step for testing purposes. removing the change.